src/components/shared/ColumnHelper.js

```diff
@@ -4,6 +4,27 @@ import Link from "../shared/Link"
 export const GENERATED_ID_COL_INDEX = 0
 export const RESTRICTED_COL_INDEX = 1
 
+const RESTRICTED_FILTER_NAMES = ["Restricted", "Not restricted"];
+export const RESTRICTED_FILTER = {
+  label: "Restricted",

```

Comment:
Can you make this "Restricted Price" or something along those lines so it's clear that this is for the price?

Reply:
Sure, no problem. I changed it to "RESTRICTED_PRICE_FILTER" and changed the labl to "Restricted price"